source/core/core.vcxproj.filters

```diff
@@ -130,8 +130,8 @@
     </ClCompile>
   </ItemGroup>
   <ItemGroup>
-    <None Include="core.natvis">
```

Comment:
We probably need to standardize on a fixed version of `premake5.exe` for both of us to use or else we will keep ping-ponging changes like this. I'm just using the most recent version made available for download on the Premake site.